docs/csharp/language-reference/toc.yml

```
@@ -34,16 +34,11 @@
           href: keywords/reference-types.md
         - name: class
           href: keywords/class.md
-        - name: delegate
-          href: keywords/delegate.md
-        - name: dynamic
-          href: keywords/dynamic.md
+        - name: Built in reference types
+          displayName: object, delegate, dynamic, string, array
```

If no addition of array info is expected:
```suggestion
          displayName: object, delegate, dynamic, string
```